elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -62,7 +76,31 @@
 
     protected QueryEngine.Transaction transaction;
 
-    public static void init() {
+    // Standard set of test queries used in dialect tests
+    protected enum TestQueryName {
+        WHERE_METRICS_ONLY,
+        WHERE_DIMS_ONLY,
+        WHERE_METRICS_AND_DIMS,
+        WHERE_METRICS_OR_DIMS,
+        WHERE_METRICS_AGGREGATION,
+        HAVING_METRICS_ONLY,
+        HAVING_DIMS_ONLY,
+        HAVING_METRICS_AND_DIMS,
+        HAVING_METRICS_OR_DIMS,
+        PAGINATION_TOTAL,
+        SORT_METRIC_ASC,
+        SORT_METRIC_DESC,
+        SORT_DIM_DESC,
+        SORT_METRIC_AND_DIM_DESC,
+        SUBQUERY,
+        GROUP_BY_DIMENSION_NOT_IN_SELECT,
```

Comment:
Not sure what this means.  the group by is in the select isn't it?

Reply:
The intent is to have a field in the GROUP BY that's not part of the SELECT. I'll double check what's going wrong with this one.

Reply:
I think it is part of the select though.

Reply:
I changed this from GROUP BY to ORDER BY, which doesn't push the field into the SELECT.